PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -75,20 +91,38 @@ def spam_call_gtta(step,numTests,node):
         response = api.get_transactions_to_approve(depth=3)
         responseVal.append(response)
         
-    responses[apiCall] = {}
-    responses[apiCall][node] = responseVal
+    world.responses[apiCall] = {}
+    world.responses[apiCall][node] = responseVal
+
+
+@step(r'getInclusionStates is called with the transaction "([^"]*)" and tips "([^"]*)" on "([^"]*)"')
+def inclusion_state_is_called(step,transaction,tips,node):
```

Comment:
Actually I am a little confused. The previous function seemed to be a general way to call *all* api methods. Why do you need also functions for specific calls.

Either have a function for specific calls or a general function.

Imo if you will adopt the gherkin table offer for the general function you will have to write less python if we add a new api call.